content/docs/dvclive/api-reference/live/end.md

```diff
@@ -0,0 +1,40 @@
+# Live.end()
+
+Signals that the current experiment has ended.
+
+```py
+def end():
+```
+
+## Usage
+
+`Live.end()` gets automatically called when exiting the context manager:
+
+```py
+from dvclive import Live
+
+with Live() as live:
+    pass
+# live.end() has been called at this point
+```
+
+It is also called when the training ends for each of the supported
+[ML Frameworks](/doc/dvclive/api-reference/ml-frameworks)
+
+## Description
+
+By default, `Live.end()` will call `Live.make_summary()` and
+`Live.make_report()`.
+
+If `save_dvc_exp=True` has been passed to `Live`, it will create a new DVC
+<abbr>experiment</abbrs>, by calling `exp save` internally, and also writing a
```

Comment:
Can `exp save` be `dvc exp save`? I don't know if they are equivalent syntax, but seems useful to have the `dvc` prefix since we aren't inside the dvc cli docs.

Reply:
It actually needs the `dvc` for the auto linker to work

Reply:
Or maybe we can skip mentioning `exp save` at all, and instead "create a new DVC experiment" can link to the `exp save` docs?